### PR TITLE
Enhancement/objects 1209

### PR DESCRIPTION
**Pull request by tcrossrfid**
### What changes were proposed in this pull request?

Cleaned up Role model

### How is this patch documented?

Code

### How was this patch tested?

Existing unit tests

#### Depends On

Nothing

### CODE REVIEW

**Comment by andreassiegel** [comment]
I think you forget to remove the now unused converters.

**Comment by tcrossrfid** [comment]
Unused converter removed.

File: src/main/java/net/smartcosmos/usermanagement/role/resource/UpdateRoleResource.java [hunk 1]
```diff
@@ -32,10 +32,10 @@
 @PreAuthorize("hasAuthority('https://authorities.smartcosmos.net/roles/update')")
 public class UpdateRoleResource {
 
-    private UpdateRoleService service;
+    private UpdateRoleServiceDefault service;
 
     @Autowired
-    public UpdateRoleResource(UpdateRoleService service) { this.service = service; }
+    public UpdateRoleResource(UpdateRoleServiceDefault service) { this.service = service; }
```
Comment (andreassiegel) [inline src/main/java/net/smartcosmos/usermanagement/role/resource/UpdateRoleResource.java]: This needs to be `UpdateRoleService`.

File: src/main/java/net/smartcosmos/usermanagement/role/resource/UpdateRoleResource.java [hunk 2]
```diff
@@ -32,10 +32,10 @@
 @PreAuthorize("hasAuthority('https://authorities.smartcosmos.net/roles/update')")
 public class UpdateRoleResource {
 
-    private UpdateRoleService service;
+    private UpdateRoleServiceDefault service;
```
Comment (andreassiegel) [inline src/main/java/net/smartcosmos/usermanagement/role/resource/UpdateRoleResource.java]: This needs to be `UpdateRoleService`

File: src/main/java/net/smartcosmos/usermanagement/role/resource/CreateRoleResource.java [hunk 3]
```diff
@@ -31,10 +31,10 @@
 @PreAuthorize("hasAuthority('https://authorities.smartcosmos.net/roles/create')")
 public class CreateRoleResource {
 
-    private CreateRoleService service;
+    private CreateRoleServiceDefault service;
```
Comment (andreassiegel) [inline src/main/java/net/smartcosmos/usermanagement/role/resource/CreateRoleResource.java]: This needs to be `CreateRoleService`.

File: src/main/java/net/smartcosmos/usermanagement/role/resource/CreateRoleResource.java [hunk 4]
```diff
@@ -31,10 +31,10 @@
...
 
     @Autowired
-    public CreateRoleResource(CreateRoleService service) { this.service = service; }
+    public CreateRoleResource(CreateRoleServiceDefault service) { this.service = service; }
```
Comment (andreassiegel) [inline src/main/java/net/smartcosmos/usermanagement/role/resource/CreateRoleResource.java]: This needs to be `CreateRoleService`.

**Pull request merged**